Missions/Missions/Games Tab/BalanceGameViewController.swift

```diff
@@ -70,7 +81,7 @@ class BalanceGameViewController: UIViewController {
     private lazy var scoreLabel: UILabel = {
         let scoreLabel = UILabel()
         scoreLabel.translatesAutoresizingMaskIntoConstraints = false
-        scoreLabel.text = "0"
+        scoreLabel.text = "\(currentScore)"
```

Comment:
Are you aware that `scoreLabel`'s text won't change depending on the `currentScore` when the game is in progress? If so, then is this fine, but because this variable is lazily instantiated, it will only set the text to 0 one time (aka at `viewDidLoad`)

Reply:
Yes, it is updated with notification center L:413-416.